src/components/form/settings.vue

```diff
@@ -25,6 +25,39 @@ except according to the terms contained in the LICENSE file.
             </i18n-t>
           </div>
         </div>
+        <div class="panel panel-simple panel-web-forms">
+          <div class="panel-heading">
+            <h1 class="panel-title">
+              <span class="badge beta">
+                {{ $t('common.beta') }}
+              </span>
+              {{ $t('webFormsSetting.webForms') }}
+            </h1>
+          </div>
+          <div class="panel-body">
+            <i18n-t tag="p" keypath="webFormsSetting.description">
+              <template #formName>
+                <strong>{{ form.nameOrId }}</strong>
+              </template>
+            </i18n-t>
+            <form id="dataset-settings-form">
+              <div class="radio">
+                <label>
+                  <input v-model="webformsEnabled" name="webformsEnabled" type="radio" :value="false"
+                    aria-describedby="dataset-setting-on-receipt" :disabled="form.awaitingResponse" @change="showConfirmation()">
```

Comment:
Now that these radio buttons just toggle the modal and don't send a request, I think they don't need `:disabled="form.awaitingResponse"`. The modal will be laid over the radio buttons, so it won't be possible for the user to interact with the radio buttons while the form request is in progress.